### DISCUSSION TITLE
Welcome to OVIWrite Discussions!

### DISCUSSION THREAD

**Discussion by MiragianCycle** [MiragianCycle/OVIWrite#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by svrvt** [comment]
Hi, Theena. I was wondering if you would like an example of integrating Zotero into NeoVim, have you noticed Benjamin's implementation of [this](https://github.com/benbrastmckie/.config)

**Comment by MiragianCycle** [comment]
Hello

I am terribly sorry that I've taken so long to reply to this email. For
some reason, this was in my spam folder.

Yes, I have seen - and been in contact with - Benjamin's config, and would
like to do something similar with oviwrite. I just haven't had the time to
sit down and integrate it.

Best
Theena

PS: Apologies for the late reply again.

On Thu, Dec 14, 2023 at 12:52 AM Rustam Uzairov ***@***.***>
wrote:

> Hi, Theena. I was wondering if you would like an example of integrating
> Zotero into NeoVim, have you noticed Benjamin's implementation of this
> <https://github.com/benbrastmckie/.config>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/MiragianCycle/OVIWrite/discussions/4#discussioncomment-7846345>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOAVIEXQDVCH6L4NY6WLC53YJH6AHAVCNFSM6AAAAABAQLWSNOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TQNBWGM2DK>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Theena Kumaragurunathan *

*M*: +94 773 017 571
*E*: ***@***.***
*W*: https://theena.net/  <https://theena.net/> |  https://miragian.com/
*L:  *https://www.linkedin.com/in/theenaklk*/*
